lib/ansible/constants.py

```diff
@@ -68,32 +69,14 @@ def set_constant(name, value, export=vars()):
     export[name] = value
 
 
+def BECOME_METHODS():
```

Comment:
I don't think switching this to a callable, will achieve the result that is expected.  People won't be calling it, so they just wouldn't get anything they expect.

Reply:
the only way to do this with the desired behaviour would be to inject a class as the constants module and have a getter for the specific value ... not sure that is somewhere i want to go.